Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -112,55 +122,55 @@ public Vector3 GetHandDelta(uint handId)
         {
             if (handId >= editorHandsData.Length)
             {
-                string message = string.Format("GetHandDelta called with invalid hand ID {0}.", handId);
+                string message = string.Format("GetHandDelta called with invalid hand ID {0}.", handId.ToString());
```

Comment:
Just curious if there is a reason to be explicit about the string conversion here? The end result should be functionally the same.